src/client/js/create.js

```diff
@@ -0,0 +1,346 @@
+
+// 배열들
+const studyType = ["면접", "프로젝트", "스터디"];
+const stackType = ["HTML", "CSS", "JavaScript", "Node.JS", "React", "Python"];
+const months = ["January", "February", "March", "April", "May", "June", "July", "August", "September", "October", "November", "December"];
+const locations = ["서울", "경기", "인천", "강원", "충북", "세종", "충남", "대전", "경북", "대구", "전남", "전북", "광주", "경남", "울산", "부산", "제주", "온라인"];
+
+// 셀렉트 박스 클릭 시 토글
+function toggleSelectBoxType(event){
+	event.preventDefault();
+	selectionsType.classList.toggle("active"); 
+}
+function toggleSelectBoxStack(event){
+	event.preventDefault();
+	selectionsLanguage.classList.toggle("active");
+}
+selectBoxType.addEventListener("click", toggleSelectBoxType);
+selectBoxStacks.addEventListener("click", toggleSelectBoxStack);
+
+// 스터디 유형 셀렉트 박스에 추가
+function addStudyType(){
+    studyType.forEach((item) => {
+        const li = document.createElement("li");
+        const button = document.createElement("button");
+        li.classList.add("stacks")
+        button.textContent = item;
+        button.classList.add("selections")
+        selectionsType.appendChild(li).appendChild(button);
+			});
+}
+addStudyType();
+
+// 스택, 기술 이름 셀렉트 박스에 추가
+function addStackType(){
+    stackType.forEach((item) => {
+        const li = document.createElement("li");
+        const button = document.createElement("button");
+        li.classList.add("languages")
+        button.textContent = item;
+        button.classList.add("selections");
+        selectionsLanguage.appendChild(li).appendChild(button);
+    });
+}
+addStackType();
+
+// 스터디 유형 셀렉트 박스 선택 시 텍스트 변경
+selectionsType.addEventListener("click", (event) => {
+  if (event.target.nodeName === "BUTTON") { 
+		selectBoxType.textContent = `${event.target.textContent}`;
+		event.preventDefault();
+	}
+  selectionsType.classList.remove("active");
+});
+
+let tagList = [];
+// 스택 유형 셀렉트 박스 선택 시 태그 추가
+selectionsLanguage.addEventListener("click", (event) => {
+	// 중복확인
+	let flag = true;
+	tagList.forEach((tag)=> { 
+		if(event.target.textContent === tag){
+			flag = false;
+		}
+	})  
+		// 확인 후 추가
+		if (flag){
+			if (event.target.nodeName === "BUTTON") {
+				const li = document.createElement("li");
+				stackTags.appendChild(li);
+				li.textContent = `${event.target.textContent}`;
+				tagList.push(event.target.textContent);
+				console.log(tagList);
+			}
+		}
+		event.preventDefault();
+    selectionsLanguage.classList.remove("active");
+});
+
+// 스택 유형 태그 제거
+stackTags.addEventListener("click", (event) => {
+	let removeTag;
+	if (event.target.nodeName === "LI") {
+		event.target.remove();
+		removeTag = tagList.indexOf(event.target.textContent);
+		tagList.splice(removeTag, 1);
+	}
+})
+
+
+// 데이트 피커 (변수들)
+let date = new Date();
+let day = date.getDate();
+let month = date.getMonth();
+let year = date.getFullYear();
+
+let selectedDate = date;
+let selectedDay = day;
+let selectedMonth = month;
+let selectedYear = year;
```

Comment:
전역에 선언한 변수들은 맨 윗쪽에 선언하는 것을 주로 선호하기는 합니다. !
참고만해주세요
이 변수는 재 할당이 되는 부분이 없어서 const로 선언을 하시면 좋을 거 같네요
```suggestion
const date = new Date();
```

Reply:
같은 맥락인데,
불변성을 지키는 부분은 상당히 중요하다고 생각합니다~~
불변성을 지켜주시면 에러가 발생해도 해당 에러에 대한 디버깅이 확실히 수월해지니까요!
생각하지도 못한 부분에 재할당이 이루어지고 있다면 그걸 찾기 위한 시간을 어마어마하게 쏟아부으셔야 합니다 ㅠ